server/tests/src/test/java/org/infinispan/server/extensions/ExtensionsIT.java

```diff
@@ -31,14 +32,7 @@ public class ExtensionsIT {
                .build();
 
    public static String[] mavenArtifacts() {
-      return new String[]{
-            "org.openjdk.nashorn:nashorn-core:15.4",
-            "org.ow2.asm:asm:9.4",
-            "org.ow2.asm:asm-analysis:9.4",
-            "org.ow2.asm:asm-commons:9.4",
-            "org.ow2.asm:asm-tree:9.4",
-            "org.ow2.asm:asm-util:9.4"
-      };
+      return Common.NASHORN_DEPS;
```

Comment:
Why is this change necessary?

Reply:
Because we already had NASHORN_DEPS in Common, and I'd rather centralize stuff